tests/run_integration_tests.sh

```diff
@@ -2,7 +2,7 @@
 
 # Check that we can connect to Postgres through PgBouncer container.
 
-pgbouncer_image=${IMAGE_NAME:-registry.devshift.net/bayesian/coreapi-pgbouncer:latest}
+pgbouncer_image=${IMAGE_NAME:-docker.io/bitnami/pgbouncer:latest}
```

Comment:
Cannot allow using of docker hub image the ci.ext won't be able to pull it please change it to some other registry .